### DISCUSSION TITLE
Beginners Guide To Making Art in Sunflower Land

### DISCUSSION THREAD

**Discussion by Myth727** [sunflower-land/sunflower-land#1466] [Art]
Looking to manage my free time more and was very interested in seeing if I can manage to make some art for the game. Can anyone provide me with all the links to get started, maybe a how to video if possible? Thanks in advance yall!

**Comment by future2back** [comment]
@Myth727 welcome to Sunflower Land.

You can start participating on this event.

**[Bumpkin Art Competition 🥳]** All necessary information is available there.

https://github.com/sunflower-land/sunflower-land/discussions/1434



and 

**[Seal Accessories | Art Request]** From Project Dignity
https://github.com/sunflower-land/sunflower-land/discussions/1442

Good luck and work 🍀 :rocket: .

**Comment by Netherzapdos** [comment]
Art Contributing guide from the Whitepaper:
https://docs.sunflower-land.com/contributing/for-designers

Info on the art style of the game:
https://docs.sunflower-land.com/contributing/game-design/visual-and-audio-styles#colour-palette

**Reply by Netherzapdos** [reply]
oh also head onto #artists-chat in the discord as there's more stoof there 🎨
